config/development.json

```diff
@@ -193,24 +192,7 @@
 		"upsell/concierge-session": true,
 		"upsell/nudge-component": true,
 		"woocommerce/extension-dashboard": true,
-		"woocommerce/extension-dashboard-stats-widget": true,
-		"woocommerce/extension-orders": true,
-		"woocommerce/extension-orders-create": true,
-		"woocommerce/extension-orders-edit": true,
-		"woocommerce/extension-products": true,
-		"woocommerce/extension-product-categories": true,
-		"woocommerce/extension-promotions": true,
 		"woocommerce/extension-referrers": true,
-		"woocommerce/extension-reviews": true,
-		"woocommerce/extension-settings": true,
-		"woocommerce/extension-settings-email": true,
-		"woocommerce/extension-settings-email-generic": true,
-		"woocommerce/extension-settings-payments": true,
-		"woocommerce/extension-settings-shipping": true,
-		"woocommerce/extension-settings-stripe-connect-flows": true,
-		"woocommerce/extension-settings-tax": true,
-		"woocommerce/extension-wcservices": true,
-		"woocommerce/extension-wcservices/international-labels": true,
 		"woocommerce/onboarding-oauth": true,
```

Comment:
`onboarding-oauth` is always true except Desktop. It enables some custom styles and copy in Woo signup and login flows. Desktop user has zero chance of getting there.